ui/js/component/common.js

```diff
@@ -22,6 +22,16 @@ export class Icon extends React.PureComponent {
   }
 }
 
+export class IconSet extends React.PureComponent {
```

Comment:
Couple issues about names:

1. It could obviously also be IconPanel, IconGroup, etc. No strong preference.
2. Do we want a name that doesn't start with icon? Makes it easy to confuse with actual icons, and I actually had to use an underscore in the CSS class because Font Awesome selects everything starting with `icon-`

Reply:
`IconSet` is fine. Please put a comment in CSS related to the break from BEM naming pattern.